simulator/risc_v/t/unit_test.cpp

```diff
@@ -155,3 +155,35 @@ TEST_CASE( "RISCV lq/sq")
     memory->load_store( &load);
     CHECK( narrow_cast<uint64>( load.get_v_dst()) == 0xf);
 }
+
+TEST_CASE( "RISCV slo")
+{
+    CHECK( RISCVInstr<uint32>( 0x20E797B3).get_disasm() == "slo $a5, $a5, $a4");
+    RISCVInstr<uint32> instr32( "slo", 0);
+    RISCVInstr<uint64> instr64( "slo", 8);
+    std::vector<std::tuple<uint32, uint32, uint32>> cases32 = {
+        {0x1C, 2, 0x73},
+        {0xFFFFFFFF, 0xAA, 0xFFFFFFFF},
+        {0xAA, 0xFF, 0x7FFFFFFF},
+        {0xAB, 0xFF, 0xFFFFFFFF},
+    };
+    std::vector<std::tuple<uint64, uint64, uint64>> cases64 = {
+        {0x1C, 2, 0x73},
+        {0xFFFFFFFF, 5, 0x1FFFFFFFFF},
+        {0xFFFFFFFFFFFFFFFF, 0xFF, 0xFFFFFFFFFFFFFFFF},
+        {0xAA, 0xFF, 0x7FFFFFFFFFFFFFFF},
+        {0xAB, 0xFF, 0xFFFFFFFFFFFFFFFF},
+    };
+    for ( const auto& c : cases32) {
+        instr32.set_v_src( std::get<0>( c), 0);
+        instr32.set_v_src( std::get<1>( c), 1);
+        instr32.execute();
```

Comment:
Our 'FuncInstr' class is not designed to run 'execute()' twice, since it writes some fields like "mask", "complete" etc. It's better to create a new object for each test.